composer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "umakantp/smarty-lint",
+    "description": "SmartyLint is a PHP tool that helps you find issues in your Smarty template files. It's designed to assist you in writing cleaner and more consistent Smarty templates",
+    "type": "project",
+    "homepage": "https://github.com/umakantp/SmartyLint",
+    "license": "BSD-3-Clause",
+    "version": "0.2.0",
```

Comment:
You should not put this. We don't want to edit this file for each release. Ref: https://getcomposer.org/doc/04-schema.md#version

Reply:
Thank you @marclaporte . I have refactored to include the suggestions